package.json

```diff
@@ -47,6 +47,7 @@
     "site": "bundle exec jekyll serve",
 
     "lint": "tslint -c tslint.json 'src/**/*.ts' 'test/**/*.ts'",
+    "lint:fix": "tslint -c tslint.json --fix 'src/**/*.ts' 'test/**/*.ts'",
```

Comment:
Hmm, since our Travis test rely on `lint` script.  

I wonder if this will actually make lint pass on travis even though the source code fails the test?